content/blog/2023/2023-03-08-code-format-with-ktlint.md

```diff
@@ -0,0 +1,125 @@
+---
+ authors: [kanake]
```

Comment:
Please add yourself to the `authors.yml` file. If you don't want to upload an author image, that's fine, but please fill out the rest.